src/javascript/EditPanelContainer/EditPanel/EditPanel.jsx

```diff
@@ -24,6 +24,10 @@ const EditPanel = ({fields, siteInfo, nodeData, formik}) => {
         };
     }, [formik.dirty]);
 
+    const onSelectLanguage = lang => {
+        console.log('Switching language to: ' + lang);
```

Comment:
why loggin that ?

Reply:
It's just temporarily, it will be implemented properly in another task.

Reply:
I propose you to add a TODO then :)
```suggestion
        // TODO BACKLOG-10541: remove this log and implement this function
        console.log('Switching language to: ' + lang);
```